db/blob/blob_source_test.cc

```diff
@@ -210,9 +217,27 @@ TEST_F(BlobSourceTest, GetBlobsFromCache) {
 
       ASSERT_FALSE(blob_source.TEST_BlobInCache(blob_file_number, file_size,
                                                 blob_offsets[i]));
+      total_bytes += bytes_read;
     }
 
+    ASSERT_EQ((int)get_perf_context()->blob_cache_hit_count, 0);
+    ASSERT_EQ((int)get_perf_context()->blob_read_count, num_blobs);
+    ASSERT_EQ((int)get_perf_context()->blob_read_byte, total_bytes);
+    ASSERT_EQ((int)get_perf_context()->getblob_read_bytes, total_bytes);
+    ASSERT_GE((int)get_perf_context()->blob_checksum_time, 0);
+    ASSERT_EQ((int)get_perf_context()->blob_decompress_time, 0);
+
+    ASSERT_EQ(statistics->getTickerCount(BLOB_DB_CACHE_MISS), num_blobs * 3);
```

Comment:
Is the `*3` needed because of the two `TEST_BlobInCache` calls in the loop? If so, it might be worth a comment :)

Reply:
Yeah. Sure.